README.md

```diff
@@ -119,19 +119,19 @@ For more information about populating database, see the [Populating the Database
 
 ### Setting up a Production Environment
 1. Clone or pull the `react-pwa-reference-storefront` repository to your directory.
-2. Navigate to the `react-pwa-reference-storefront` directory.<br>
+2. Navigate to the `react-pwa-reference-storefront` directory.
+3. Add any custom configurations to the Dockerfile, Docker Compose file, and Nginx Configuration file
+    1. Provide any custom configurations you would like passed into your Docker container at runtime directly below the existing `RUN jq` command. One has bee provided to you as an example. Be sure to maintain the prefix of `NGINX_REPLACE_` for your custom configuration.
+    **Note:** You must use jq syntax for the command. More information on jq is availble [here](https://stedolan.github.io/jq/manual/).
+    2. Update the `./docker/prod/nginx.conf` file to include the new configuration directly below the existing one by adding a new `sub_filter` statement with your configuration within the configuration block. Be sure to expose this value with a more generic value, which will be used for the environment variable in the next step.
+    3. Update the `./docker/prod/docker-compose.yaml` file to include your environment configuration variable from step 2 into the `environment` block, and set it to the value you desire to pass into your container.
 3. Build a Production Docker Image:
-    1. Clone or pull the `react-pwa-reference-storefront` repository to your directory.
-    2. Run the `cd react-pwa-reference-storefront` command.
-    3. Run the `docker build -t ep-store -f ./docker/prod/Dockerfile .` command.
-    4. Push the `ep-store` image to the docker repository.
-
+    1. Run the `docker build -t ep-store -f ./docker/prod/Dockerfile .` command.
+    2. Push the `ep-store` image to the docker repository.
 4. Run a Production Docker Image:
     1. Navigate to the `docker/prod/` directory.<br>
-    2. Copy the following files to the user home directory on the remote host:<br>
-        * `docker-compose.yaml`
-        * `nginx.conf`
-    3. In the `nginx.conf` file, update the following parameters:
+    2. In the `docker-compose.yaml` file, update the following parameters:
+        * `$DOCKER_REPO` with `ep-store`.
```

Comment:
Use a table here

Parameter                             Value 
`$DOCKER_REPO`              `ep-store`
`$CORTEX_URL`                  The Cortex server URL.
`$AUTHSERVICE_URL`        The Cortex Account Management service URL. (This  parameter is required for all scenarios. For a B2C storefront, set this value to the`$CORTEX_URL` parameter).

`$DOMAIN`                             The domain name without `http://`.       


use table for all the parameters